src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -155,10 +155,334 @@ public SyntaxNode MethodDeclaration(IMethodSymbol method, IEnumerable<SyntaxNode
         }
 
         /// <summary>
-        /// Creates a method declaration.
+        /// Creates an implicit conversion declaration.
+        /// </summary>
+        public virtual SyntaxNode ImplicitConversionDeclaration(
+            IEnumerable<SyntaxNode> parameters = null,
+            SyntaxNode returnType = null,
+            Accessibility accessibility = Accessibility.NotApplicable,
+            DeclarationModifiers modifiers = default(DeclarationModifiers),
+            IEnumerable<SyntaxNode> statements = null)
+        {
+            throw new NotImplementedException();
```

Comment:
Missed that the docs for NotImplementedException call this out as an official use case, so the type is probably OK, but I still think NotSupported is better and there is precedent in mscorlib for that. +1 on adding an actionable message either way.

https://github.com/dotnet/coreclr/search?utf8=%E2%9C%93&q=NotSupported_SubclassOverride

https://github.com/dotnet/coreclr/blob/29f52c442a08240b3df1c5893901f9dfcae6751f/src/mscorlib/src/mscorlib.txt#L1379